lib/protobuf/protoc/frame/parser.ex

```diff
@@ -65,23 +65,29 @@ defmodule Protobuf.Protoc.Parser do
     file
   end
 
-  @spec parse_msg(Context.t(), String.t(), Google.Protobuf.DescriptorProto.t()) :: any
-  defp parse_msg(%{namespace: ns} = ctx, file_fqn, desc) do
-    fqn = Utils.join_name([file_fqn, desc.name])
+  @spec parse_msg(Context.t(), String.t(), String.t(), Google.Protobuf.DescriptorProto.t()) :: any
+  defp parse_msg(_ctx, _file_fqn, _parent_msg, []), do: []
+
+  defp parse_msg(%{namespace: ns} = ctx, file_fqn, parent_msg_fqn, desc) do
```

Comment:
I think we can change `file_fqn, parent_msg_fqn` to a single `parent_fqn`?

Reply:
I thought we can get all messages(include nested messages) in a file if we have the file_fqn in message metadata. I can remove it if it's not useful.